examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -75,7 +75,7 @@ git submodule update --init external/fv3gfs-util external/daint_venv
 
 # set GT4PY version
 cd $ROOT_DIR
-export GT4PY_VERSION=`grep "GT4PY_VERSION=" docker/Makefile.image_names | cut -d '=' -f 2`
+export GT4PY_VERSION=`grep "GT4PY_VERSION ?=" docker/Makefile.image_names | cut -d '=' -f 2`
```

Comment:
`export GT4PY_VERSION=`grep "GT4PY_VERSION" docker/Makefile.image_names  | tr -d ' ' | cut -d '=' -f 2`

Reply:
thanks, used that with a -m 1